tesseract/tesseract_planning/tesseract_process_managers/src/process_input.cpp

```diff
@@ -120,4 +120,23 @@ std::size_t ProcessInput::size()
   return 0;
 }
 
+void ProcessInput::setStartInstruction(Instruction instruction)
+{
+  start_instruction = instruction;
+  start_instruction_ptr = &start_instruction;
```

Comment:
This may break things, because now they both exist and I believe there is logic that checks one or the other.